static/app/views/performance/utils.tsx

```diff
@@ -34,13 +40,15 @@ const BACKEND_PLATFORMS: string[] = [...backend];
 const MOBILE_PLATFORMS: string[] = [...mobile];
 
 export function platformToPerformanceType(
-  projects: Project[],
+  projects: (Project | ReleaseProject)[],
   projectIds: readonly number[]
 ) {
   if (projectIds.length === 0 || projectIds[0] === ALL_ACCESS_PROJECTS) {
     return PROJECT_PERFORMANCE_TYPE.ANY;
   }
-  const selectedProjects = projects.filter(p => projectIds.includes(parseInt(p.id, 10)));
+  const selectedProjects = projects.filter(p =>
+    projectIds.includes(parseInt(`${p.id}`, 10))
```

Comment:
Isn't p.id already a string?

Reply:
ah it is for the `Project` type, but `ReleaseProject.id` is a `number`, and typescript was complaining here about type`number` is not assignable to type `string`